docker-compose-services/drupalci-chromedriver/docker-compose.chromedriver.yaml

```
@@ -17,4 +17,5 @@ services:
       # - SIMPLETEST_DB=mysql://db:db@db:3306/db
       - SIMPLETEST_BASE_URL=http://web
       - BROWSERTEST_OUTPUT_DIRECTORY=/tmp
+      - BROWSERTEST_OUTPUT_BASE_URL=http://${DDEV_SITENAME}.ddev.site
```

I think this would be more likely to work with $DDEV_PRIMARY_URL, please try it out and if it's good, you can commit here with the button and I'll pull it.
```suggestion
      - BROWSERTEST_OUTPUT_BASE_URL=${DDEV_PRIMARY_URL}
```

---

Just tested and that works great. Thanks for the suggestion @rfay.